### PR TITLE
Develop the `<Table />` component

### PR DESCRIPTION
**Pull request by Andresbl123**
In apps we will need a visual structure to render a bunch of data in a table. Analyse the custom component `<Table />` developed by the Linpar team, propose (if it is considered correct) a refactoring of this component and finally develop and publish a version of the component design system.

### CODE REVIEW

File: src/components/data/Table/interface.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      ))}
+    </>
+  ) : (
+    <StyledTd>
+      <MdOpenInNew />
+    </StyledTd>
+  );
+}
+
+function TableUI(props) {
```
Comment (cmarin001) [inline src/components/data/Table/interface.jsx:68]: por favor revisar y corregir los errores que arroja storybook
image

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/components/data/Table/stories/Default.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+    },
+    {
+      id: "14",
+      username: "Johan Nova",
+      code: "JNOVA",
+      userID: "589647",
+      position: "Adviser",
+    },
+  ],
+  entriesPagination: [
```
Comment (wfercanas) [inline src/components/data/Table/stories/Default.stories.jsx:118]: Esta propiedad no tiene sentido porque no es una prop del componente.

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/components/data/Table/Pagination/styles.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
+import { colors } from "../../../../shared/colors/colors";
+
+const StyledContentPagination = styled.div`
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/components/data/Table/Pagination/styles.js]: Resuelto

File: src/components/data/Table/Pagination/styles.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  display: flex;
+  flex-direction: row;
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/components/data/Table/Pagination/styles.js]: Resuelto

File: src/components/data/Table/Pagination/styles.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  justify-content: flex-end;
+  align-items: center;
+  height: 52px;
+  width: 100%;
+`;
+
+const StyledContentButtons = styled.div`
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/components/data/Table/Pagination/styles.js]: Resuelto

File: src/components/data/Table/Pagination/styles.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    height: 16px;
+    color: ${colors.sys.text.dark};
+  }
+
+  & svg:hover {
+    color: ${colors.sys.text.information};
+  }
+`;
+
+const StyledContentText = styled.div`
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/components/data/Table/Pagination/styles.js]: Resuelto

File: src/components/data/Table/stories/Default.stories.jsx (hunk 2 above)
Comment (Andresbl123) [reply src/components/data/Table/stories/Default.stories.jsx]: resuelto

**Comment by cmarin001** [comment]
tenemos un error cuando se abre la historia `Pagination`
image

**Review by wfercanas — APPROVED** [review]

**Pull request merged**